content/solana-nft.md

```diff
@@ -0,0 +1,637 @@
+```
+
+![Gif of Whitelist Token](../assets/solana-nft-whitelist-token.gif)
+
+If you would like to transfer whitelist tokens to another wallet address, use the following command:
+
+```sh
+spl-token transfer <WHITELIST_TOKEN_ADDRESS> 1 <WALLET_ADDRESS> --fund-recipient
+```
+
+![Gif of Whitelist Token Transfer](../assets/solana-nft-whitelist-token-transfer.png)
+
+Lastly, let's set our connection to Devnet back to Metaplex RPC:
+
+```sh
+solana config set --url https://metaplex.devnet.rpcpool.com/
+```
+
+### 10. Enable Whitelist
+
+Let's update our Candy Machine to enable whitelist settings.
+
+First, open the `config.json` file and reset the `gatekeeper` field to `null`.
+
+```json
+    "gatekeeper": null
+```
+
+Update the `goLiveDate` to sometime in the future:
+
+```json
+    "goLiveDate": "25 Dec 2022 00:00:00 GMT"
+```
+
+Update `whitelistMintSettings`
+
+- `burnEveryTime: true` will burn the whitelist token when minting an NFT
+- `mint` specifies the token mint of the whitelist token
+- `presale: true` setting allows whitelist token holders to mint before the `goLiveDate`
+- `discountPrice` specifies a discounted mint price for whitelist token holders
+
+```json
+    "whitelistMintSettings": {
+        "mode": { "burnEveryTime": true },
+        "mint": "<WHITELIST_TOKEN_ADDRESS>",
+        "presale": true,
+        "discountPrice": 0.01
+    }
+```
+
+Update the Candy Machine by running the `update_candy_machine` command again:
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts update_candy_machine \
+    -e devnet \
+    -k ~/.config/solana/devnet.json \
+    -cp config.json \
+    -c example
+```
+
+Refresh [localhost:3000](http://localhost:3000) in the browser and mint from a wallet that holds the whitelist token:
+
+![Gif of Whitelist Mint](../assets/solana-nft-whitelist-mint.gif)
+
+Go ahead and mint out the Candy Machine from a wallet with the whitelist token until you see the following:
+
+![Screenshot of Sold Out UI](../assets/solana-nft-sold-out.png)
+
+### 11. Withdraw Rent
+
+Now that our Candy Machine is fully minted, the rent used for the Candy Machine can be retrieved by running the `withdraw` command:
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts withdraw <candy_machine_id> \
+    -e devnet \
+    -k ~/.config/solana/devnet.json
+```
+
+![Gif of Withdraw](../assets/solana-nft-metaplex-withdraw.gif)
+
+### 12. Signing NFTs
+
+Finally, sign the NFTs to verify yourself as the creator of the collection by running the `sign_all` command:
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts sign_all \
+    -e devnet \
+    -k ~/.config/solana/devnet.json \
+    -c example
+```
+
+![Gif of Sign All](../assets/solana-nft-metaplex-sign-all.gif)
+
+# Challenge
+
+Now it’s your turn to build something independently. Create a new Candy Machine using your own images, metadata, and configuration.
+
+Recall that the sequence of steps are as follows:
+
```

Comment:
I feel like we need to hammer these steps in the Overview as well. The overview doesn't seem to match the demo as well as I'd like it to. It's got great explanation, but the flow doesn't necessarily match. Does that make sense?

Reply:
@jamesrp13 I'm not sure we grok the wording flow you are going for...  after reviewing @ZYJLiu 's updates... do you have any suggestions - would it make any sense to have the list in the Challenge match the Deemo's #'d section headers?